### ISSUE TITLE
module member of __import__("pkgutil") not recognized

### ISSUE THREAD

**Issue by roeap** [microsoft/pyright#299]
**Describe the bug**
I try to follow Microsofts suggestions on how to build namespace packages, which [requires](https://github.com/Azure/azure-sdk-for-python/blob/master/doc/dev/packaging.md#what-do-i-do-in-my-files-to-achieve-that) the following line in `__init__.py` files to be present. 

```py
__path__ = __import__('pkgutil').extend_path(__path__, __name__)
``` 
Which in versions `1.0.77` and `1.0.79` raises the following issue.

![image](https://user-images.githubusercontent.com/42610831/67268344-7d994c80-f4b4-11e9-93b2-68edc1f7eb6e.png)

**To Reproduce**
create a module folder, and place an `__init__.py` file with the line above in it.

**Expected behavior**
Pyright should recognize the `extend_path` function as valid member, also given the above method of importing / referencing the module.

**Screenshots or Code**
see above.

**VS Code extension or command-line**
VS Code Extension version 1.0.79

**Additional context**
thanks for this awesome tool! Our code is much better for it!

**Comment by erictraut** [comment]
Thanks for the bug report. I'll think about how best to address this.

In the meantime, you can use a "# type: ignore" comment on that line or at the top of the file to suppress the error.

**Comment by erictraut** [comment]
This is now fixed in 1.0.80, which I just published.

**Comment by roeap** [comment]
Great! Thanks for the very rapid resolution!
